soccer/gameplay/plays/restarts/basic_indirect.py

```diff
@@ -0,0 +1,53 @@
+import standard_play
+import behavior
+import skills.move
+import skills.pivot_kick
+import constants
+import robocup
+import math
+import main
+import tactics.coordinated_pass
+import evaluation.passing_positioning
+import enum
+
+
+class BasicIndirect(standard_play.StandardPlay):
+
+    def __init__(self, indirect=None):
+        super().__init__(continuous=True)
+
+        self.add_transition(behavior.Behavior.State.start,
+                            behavior.Behavior.State.running, lambda: True,
+                            'immediately')
+
+        # FIXME: this could also be a PivotKick
+        kicker = skills.line_kick.LineKick()
+        kicker.use_chipper = True
+
+        kicker.kick_power = 50
+
+        receive_pt = robocup.Point(0, 3 * constants.Field.Length / 4)
+        
+        pass_behavior = tactics.coordinated_pass.CoordinatedPass(
+            receive_pt,
+            None,
+            (kicker, lambda x: True),
+            receiver_required=False,
+            kicker_required=False,
+            prekick_timeout=9)
+        self.add_subbehavior(pass_behavior, 'kicker')
```

Comment:
This name should probably be changed to represent it's a pass not a pivot/line kick